fs/sbin/geomesa-entrypoint.sh

```diff
@@ -8,13 +8,12 @@ ROLE=${1:-}
 ACCUMULO_USER=${ACCUMULO_USER:-root}
 
 if [ $ROLE = "register" ]; then
-  wait_until_accumulo_is_available
-  accumulo shell -u ${ACCUMULO_USER} -p ${ACCUMULO_PASSWORD} -e \
-    "createnamespace geomesa"
-  accumulo shell -u ${ACCUMULO_USER} -p ${ACCUMULO_PASSWORD} -e \
-    "config -s general.vfs.context.classpath.geomesa=file:///opt/geomesa/accumulo/[^.].*.jar"
-  accumulo shell -u ${ACCUMULO_USER} -p ${ACCUMULO_PASSWORD} -e \
-    "config -ns geomesa -s table.classpath.context=geomesa"
+  wait_until_accumulo_is_available ${INSTANCE_NAME}
+  HDFS_GEOMESA_RUNTIME="/${INSTANCE_NAME}-classpath"
```

Comment:
This would conflict if geowave and geomesa shared an HDFS instance, with both iterators in the same HDFS path.